### ISSUE TITLE
Recreate repo as GitHub fork?

### ISSUE THREAD

**Issue by paulreimer** [espressif/esp-nimble#1]
I notice this repo is a fork of recent `mynewt` master, but the commit history is different.

Maybe you would consider re-creating this repo using the GitHub Fork feature, to keep the history?
(it might make it easier for Espressif to merge upstream changes, and easier for us to see what changes have been made)

**Comment by dhrishi** [comment]
@paulreimer As you mentioned, esp-nimble repository is a fork of mynewt-nimble master (last synced around 20 days back). I don't understand when you say that the commit history is different.

The latest mynewt-nimble commit in this repository is: 137efe3bc2551734100c1ded1031b6cd3b0d18bf and the same commit id can be seen in the original master repository too. Let me know in case I am missing something.

**Comment by paulreimer** [comment]
Right, sorry about that. You are right, seems like `git` is fine, I added the upstream repo as an alternate remote, it's the same history line.

I suppose the only thing I'm missing is the GitHub integration (PRs from this repo to upstream Nimble/other forks, or doing diffs via the GitHub web UI), and since it makes it easier to share PRs it inspires that ESP32/FreeRTOS support might get officially merged to upstream some time in the future.
